src/server/common/controller/generic-page-controller/index.js

```diff
@@ -63,13 +69,21 @@ export default class GenericPageController {
     const sendMetric = this.page.reportMetrics?.[method]?.[event]
 
     if (sendMetric) {
-      this.metrics.putMetric(
-        `${method}::${event}-${this.page.urlPath}`,
-        1,
-        Unit.Count,
-        StorageResolution.Standard
-      )
+      this.logger.info(`${method}::${event}-${this.page.urlPath}`)
+    }
+  }
+
+  /**
+   *
+   * @param {string} field
+   * @param {string} error
+   */
+  sendErrorMetric(field, error) {
+    if (!config.get('isProduction')) {
+      return
     }
+
+    this.logger.info(`${field} errored:${error}-${this.page.urlPath}`)
```

Comment:
Could we make this more human readable? 

e.g. `User encountered a validation error on ${this.page.urlPath}, on the ${field} field : ${error}`?